app/companies/search-talents/talent-result.tsx

```diff
@@ -29,8 +29,8 @@ export default function TalentResult({ talents }: { talents: Talent[] }) {
             image={talent.imageUrl}
             countryFlag="/img/country_flag.png" // TODO: create flag table
             city={talent.city}
-            rate={Number(talent.rate)}
-            currency={talent.currency}
+            rate={talent.rate}
+            currency="GHV" //TODO : modify format of {talent.currency}
```

Comment:
`// TODO: modify format of {talent.currency}` it's a convention comment.

We must follow the conventions, spaces included.